pkg/tss/upload/out/etc/group

```diff
@@ -0,0 +1,6 @@
+root:x:0:root
+bin:x:1:root,bin,daemon
+daemon:x:2:root,bin,daemon
+sys:x:3:root,bin,adm
+adm:x:4:root,adm,daemon
+tss:x:116:
```

Comment:
generally we do adduser etc in the container rather than adding group and passwd files.